Brokkr/src/main/include/commands/MoveArmIntake.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc2/command/CommandHelper.h>
+#include <frc2/command/ParallelDeadlineGroup.h>
+
+#include "subsystems/Arm.h"
+#include "subsystems/Claw.h"
+#include "subsystems/Wrist.h"
+
+class MoveArmIntake
+    : public frc2::CommandHelper<frc2::ParallelDeadlineGroup,
+                                 MoveArmIntake> {
+ public:
+  MoveArmIntake(Arm& arm, Claw& claw, double armAngle);
+
+ private:
+ Arm& mArm;
+ Claw& mClaw;
+ double mArmAngle;
```

Comment:
Remove all three of the member variables. This command passes all of the parameters to other classes, so the parameters do not need to be held by this class.